tests/fixtures/redis/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "init" {
```

Comment:
Should we omit this variable, as it's not used? And just rely on testing against a live init module? Cc @Spikra